Assets/Scripts/Puzzle/Lever.cs

```diff
@@ -21,32 +24,52 @@ public class Lever : SignalEmitter {
 			if (leverTimer >= timerEnd)
 				Deactivate();
 		}
+
+		if (!hasStartedFastTimeSound && leverType == LeverType.Timed   // Should this maybe be reordered to test for the leverType first instead? maybe it acts like a boolean?
+								&& leverTimer >= fastTimeAudioStart) {
```

Comment:
These are just simple comparisons and bool checks which are fast, no need to reorder.
```suggestion
		if (!hasStartedFastTimeSound && leverType == LeverType.Timed && leverTimer >= fastTimeAudioStart) {
```